products/jbrowse-react-linear-genome-view/README.md

```diff
@@ -29,7 +29,7 @@ function View() {
 }
 ```
 
-For a full working example, see [this example](docs/example.md).
+For a full working example, see [this example](docs/example.md) or a demo here
```

Comment:
Is there meant to be a link on "here"?

Reply:
this was a loose end while editing. My aim was to give the users an actual working CRA example for example that they could clone and use perhaps

Reply:
could remove if we think it's unneeded, but seems like it would just ensure that a user can get setup

Reply:
went ahead and removed for now